website/docs/providers/proxy/forward_auth.mdx

```diff
@@ -26,131 +21,10 @@ This mode differs from the _Forward auth (single application)_ mode in the follo
 -   You don't have to configure an application in authentik for each domain
 -   Users don't have to authorize multiple times
 
-There are however also some downsides, mainly the fact that you **can't** restrict individual
-applications to different users.
+There are however also some downsides, mainly the fact that you **can't** restrict individual applications to different users.
 
 The only configuration difference between single application and domain level is the host you specify.
```

Comment:
I'd suggest that since this content about configuration differences refers to both single-app and domain-level, we move it up above the two sections. But let's merge this PR first, and I can do a separate PR to move this section.